website/docs/docs/dbt-cloud/on-premises/prerequisites.md

```diff
@@ -71,9 +71,9 @@ dbt Cloud can send email notifications when runs finish. You can provide a set o
 
 ### Github
 
-dbt Cloud can integrate with your Github Enterprise installation to enable interacting with Github Pull Requests, as well as automatically permissioning your environments via Github's API. To set this up, you will need to be an admin on your Github Enterprise organization.
+dbt Cloud can integrate with your Github, Github Enterprise, or Github Enterprise Server installation to enable interacting with Github Pull Requests, as well as automatically permissioning your environments via Github's API. To set this up, you will need to be an admin on your Github Enterprise organization.
```

Comment:
This might be too much but I have often gotten the question of how do I tell is I'm an admin. Does it makes sense to answer that question here?

Reply:
@amychen1776 yes -- i've fixed the language (it's an Owner, not an admin), and linked out to the Github docs around org permissions